app/assets/stylesheets/components/fixed_table.scss

```diff
@@ -0,0 +1,22 @@
+.FixedTable {
+  table {
+    table-layout: fixed;
+    border-collapse: collapse;
+
+    td {
+      padding: 0;
+    }
+
+    tr.clickable-row:hover {
```

Comment:
This is the actual diff for the styling - it's my mistake mixing the refactoring work together with the style change, which makes it difficult to see what actually changed.